photon-core/src/main/java/org/photonvision/vision/pipe/impl/UncropApriltagsPipe.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) Photon Vision.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.photonvision.vision.pipe.impl;
+
+import edu.wpi.first.apriltag.AprilTagDetection;
+import java.util.ArrayList;
+import java.util.List;
+import org.opencv.core.Rect;
+import org.photonvision.vision.pipe.CVPipe;
+
+public class UncropApriltagsPipe
+        extends CVPipe<List<AprilTagDetection>, List<AprilTagDetection>, Rect> {
+
+    public UncropApriltagsPipe(int width, int height) {
+        this.params = new Rect(0, 0, width, height);
+    }
+
+    @Override
+    protected List<AprilTagDetection> process(List<AprilTagDetection> in) {
+        List<AprilTagDetection> temp = new ArrayList<AprilTagDetection>();
+
+        for (AprilTagDetection detection : in) {
+            temp.add(
+                    new AprilTagDetection(
+                            detection.getFamily(),
+                            detection.getId(),
+                            detection.getHamming(),
+                            detection.getDecisionMargin(),
+                            detection.getHomography(),
```

Comment:
Will the homography matrix need to change when you un-offset the detection? Is homography used by AprilTagPoseEstimatorPipe, which calls into `estimatePoseOrthogonalIteration`?

Reply:
Homography will need to be updated. See https://github.com/AprilRobotics/apriltag/blob/master/apriltag_pose.c#L509C5-L509C37